Branch-SDK/src/main/java/io/branch/referral/Branch.java

```diff
@@ -2558,7 +2558,11 @@ public void onAppLinkFetchFinished(String nativeAppLinkUrl) {
         if (getInitState() == SESSION_STATE.UNINITIALISED || !isFirstInitialization) {
             registerAppInit(initRequest, false);
         } else {
-            PrefHelper.Debug("Warning: consecutive session initialization detected! Use reInitSession() instead.");
+            PrefHelper.Debug("Warning: consecutive session initialization blocked!");
+            if (callback != null) {
+                callback.onInitFinished(null,
+                        new BranchError("Session initialization blocked.", ERR_BRANCH_ALREADY_INITIALIZED));
```

Comment:
"blocked" sounds inconsistent with other error messages.  Semantically the session initialization "failed", because it is "ALREADY_INITIALIZED".

Reply:
Changed to "failed".